src/webview/columns/ascii-column.tsx

```diff
@@ -32,13 +34,28 @@ export class AsciiColumn implements ColumnContribution {
     readonly id = 'ascii';
     readonly label = 'ASCII';
     readonly priority = 3;
+
     render(range: BigIntMemoryRange, memory: Memory, options: TableRenderOptions): ReactNode {
         const startOffset = toOffset(memory.address, range.startAddress, options.wordSize);
         const endOffset = toOffset(memory.address, range.endAddress, options.wordSize);
-        let result = '';
-        for (let i = startOffset; i < endOffset; i++) {
-            result += getASCIIForSingleByte(memory.bytes[i]);
+
+        const indices = Array.from({ length: endOffset - startOffset }, (_, a) => a + startOffset);
+        const chunks = chunkIntoN(indices, options.groupsPerRow);
+        const groups: ReactNode[] = [];
+
+        for (const chunk of chunks) {
+            let result = '';
+            for (const i of chunk) {
+                result += getASCIIForSingleByte(memory.bytes[i]);
+            }
+
+            groups.push(<span key={`ascii_${chunk[0]}-${chunk[chunk.length - 1]}`}>{result}</span>);
```

Comment:
I'm not sure how much value we get out of all the extra arrays created here for bookkeeping. Something like this should give the same result:

```suggestion
        const bytesPerGroup = options.wordSize * options.wordsPerGroup / 8;
        const groups = [];
        let groupAscii = '';
        for (let i = startOffset; i < endOffset; i++) {
            groupAscii += getASCIIForSingleByte(memory.bytes[i]);
            if (i + 1 === endOffset || groupAscii.length === bytesPerGroup) { // Could do (i - startOffset + 1) % bytesPerGroup === 0 if we want to be agnostic about the length ASCII for each byte.
                groups.push(<span key={`ascii_${i - bytesPerGroup + 1}-${i}`}>{groupAscii}</span>);
                groupAscii = '';
            }
```